src/Plot.tsx

```diff
@@ -55,6 +55,7 @@ export default function Plot({
   margin = {},
   colorScheme,
   children,
+  viewPortStyle = {},
```

Comment:
```suggestion
  viewportStyle = {},
```

"viewport" is a single word